packages/moonstone/Scrollable/ScrollButtons.js

```diff
@@ -279,23 +307,33 @@ class ScrollButtons extends Component {
 
 	onKeyDownPrev = (ev) => {
 		const
+			{focusableScrollbar} = this.props,
 			{nextButtonDisabled} = this.state,
 			{keyCode} = ev;
 
-		if (isPageDown(keyCode) && !nextButtonDisabled) {
-			Spotlight.focus(this.nextButtonNodeRef);
+		if (isPageDown(keyCode)) {
+			if (focusableScrollbar && !nextButtonDisabled) {
+				Spotlight.focus(this.nextButtonNodeRef);
+			} else if (!focusableScrollbar) {
+				this.onClickNext(ev);
+			}
 		} else if (isPageUp(keyCode)) {
 			this.onClickPrev(ev);
 		}
 	}
 
 	onKeyDownNext = (ev) => {
 		const
+			{focusableScrollbar} = this.props,
 			{prevButtonDisabled} = this.state,
 			{keyCode} = ev;
 
-		if (isPageUp(keyCode) && !prevButtonDisabled) {
-			Spotlight.focus(this.prevButtonNodeRef);
+		if (isPageUp(keyCode)) {
+			if (focusableScrollbar && !prevButtonDisabled) {
+				Spotlight.focus(this.prevButtonNodeRef);
+			} else if (!focusableScrollbar) {
+				this.onClickPrev(ev);
```

Comment:
Do we have a requirement to scroll up in this case?

Reply:
I just implemented it as Enyo VirtualList and Scroll do.